graylog2-web-interface/webpack.config.js

```diff
@@ -152,7 +150,6 @@ if (TARGET === 'build') {
           except: ['$super', '$', 'exports', 'require'],
         },
       }),
-      new webpack.optimize.DedupePlugin(),
```

Comment:
In the PR description, you mention that the `DedupePlugin` doesn't work with the changes: what are the implications? And, if we still need it, can we fix the issue so it works again?